snapshots/devmapper/config.go

```diff
@@ -82,6 +85,10 @@ func (c *Config) parse() error {
 		return errors.Wrapf(err, "failed to parse base image size: '%s'", c.BaseImageSize)
 	}
 
+	if c.FileSystemType == "" {
+		c.FileSystemType = "ext4"
+	}
```

Comment:
```suggestion
	switch c.FileSystemType {
	case string(fsTypeExt4), string(fsTypeXfs):
	case "":
		c.FileSystemType = string(fsTypeExt4)
	default:
		return fmt.Errorf("unexpected FileSystemType: %q", c.FileSystemType)
	}
```

Reply:
Thanks for the suggestion. Will use this one.